keyboards/wilba_tech/rama_works_m60_a/keymaps/mguterl/keymap.c

```diff
@@ -0,0 +1,82 @@
+#include QMK_KEYBOARD_H
+
+
+#define QWER 0
+#define GAMING 1
+#define HHKB 2
+#define UTIL 3
+#define LIGHTING 4
```

Comment:
Lines 4-8 should be replaced with:

```c
enum layer_names {
    QWER,
    GAMING,
    HHKB,
    UTIL,
    LIGHTING,
};
```

This is identical to the set of defines you have here.